### PR TITLE
kie-server-tests: Use Standalone controller in Kie Server Controller …

### PR DESCRIPTION
**Pull request by jakubschwan**
…integration tests

### CODE REVIEW

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-controller/pom.xml [hunk 1]
```diff
@@ -16,23 +16,23 @@
     <dependencies>
```
Comment (sutaakar) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-controller/pom.xml]: kie-server-controller-standalone version should be available from kie bom, this dependencyManagement section could be deleted.

**Comment by sutaakar** [comment]
Please also delete kie-server-controller-test-war as it is not needed any more.

**Comment by jakubschwan** [comment]
Standalone controller can't be deploy on Tomcat 8 - this issue is reported as [JBPM-6012](https://issues.jboss.org/browse/JBPM-6012)

**Comment by mswiderski** [comment]
@jakubschwan I believe it can be replaced with this pr https://github.com/kiegroup/droolsjbpm-integration/pull/992, could you please close this one if that's the case?

**Comment by jakubschwan** [comment]
Close this PR, changes are included in #992

**Pull request closed without merging**